timely_beliefs/tests/test_belief_query.py

```diff
@@ -56,13 +56,13 @@ def test_query_belief_for_sensor_with_unique_knowledge_time(
 
 
 @pytest.fixture(scope="function")
```

Comment:
Probably, this fixture should go in conftest.py.

Reply:
The beliefs data set up here is only used in the scope of this module, so I think it's the right place. Fixtures don't *have* to be in conftests per se.